src/app/app.component.ts

```diff
@@ -0,0 +1,10 @@
+import { Component } from '@angular/core';
+     
+@Component({
+    selector: 'app-comp',
+    templateUrl: './templates/app.component.html',
+    styleUrls: ['./styles/app.component.css']
```

Comment:
for styles we usually don't use separate folder, let's use refer for styles file same as for template and keep it in same folder

Reply:
I mean, you don't need to move styles and templates to separate folders, you can keep them in same place with app.component.ts